.github/workflows/ci.yaml

```diff
@@ -16,7 +16,7 @@ jobs:
 
       - uses: actions/setup-dotnet@v4
         with:
-          dotnet-version: "6.0.x"
+          dotnet-version: "8.0.x"
```

Comment:
If you omit this parameter, the task reads the `global.json` file in the root folder to get the version. Is there a reason to specify it?

https://github.com/actions/setup-dotnet?tab=readme-ov-file#usage

Reply:
ah good to know, yeah guess we could remove it given the `global.json` should be kept up to date